includes/admin/class-omise-admin-page.php

```diff
@@ -29,6 +29,9 @@ public function add_message( $type, $message ) {
 			case 'message':
 				$this->messages[] = $message;
 				break;
+			default:
+				$this->messages[] = $message;
+				break;
```

Comment:
can we remove the case `message`? or is it possible to do this? since the output is the same
```suggestion
			case 'message':
			default:
				$this->messages[] = $message;
				break;
```

Reply:
Yeah we can. I will remove it.